File: BeerViewer/Models/Raw/kcsapi_charge.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BeerViewer.Models.Raw
{
	public class kcsapi_charge
	{
		public kcsapi_charge_ship[] api_ship { get; set; }
		public int[] api_material { get; set; }
		public int api_use_bou { get; set; }
	}
	public class kcsapi_charge_ship
	{
		public int api_id { get; set; }
		public int api_fuel { get; set; }
		public int api_bull { get; set; }
		public int[] api_onslot { get; set; }
	}
}

File: WindowFrame/source/System/Interfaces/kcsapi_charge.ts
interface kcsapi_charge {
    api_ship: kcsapi_charge_ship[];
    api_material: number[];
    api_use_bou: number;
}
interface kcsapi_charge_ship {
    api_id: number;
    api_fuel: number;
    api_bull: number;
    api_onslot: number[];
}